packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -150,6 +150,20 @@ export class MarketOperationUtils {
             quoteSourceFilters.isAllowed(ERC20BridgeSource.Balancer),
         );
 
+        const {
+            onChain: sampleCreamOnChain,
+            offChain: sampleCreamOffChain,
+        } = this._sampler.creamPoolsCache.howToSampleCream(
+            takerToken,
+            makerToken,
+            quoteSourceFilters.isAllowed(ERC20BridgeSource.Cream),
+        );
+
+        const excludedSources = [
```

Comment:
This name is a bit confusing as there is an `opts.excludedSources`.

It's probable duplicated in this file twice as well possibly? Could use a helper

Reply:
Fixed naming, I think it's straightforward enough to leave the code the same for now.